wgengine/router_openbsd.go

```diff
@@ -5,25 +5,26 @@
 package wgengine
 
 import (
+	"bytes"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net"
+	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
 
 	"github.com/tailscale/wireguard-go/device"
 	"github.com/tailscale/wireguard-go/tun"
 	"github.com/tailscale/wireguard-go/wgcfg"
+	"tailscale.com/atomicfile"
 	"tailscale.com/types/logger"
 )
 
-// For now this router only supports the userspace WireGuard implementations.
-//
-// There is an experimental kernel version in the works:
+// For now this router only supports the WireGuard userspaces implementations in
+// the BSDs. There is an experimental kernel version in the works for OpenBSD:
```

Comment:
I would drop "in the BSDs" and just end the sentence with "implementation".  (also singular)